pygmt/src/grd2cpt.py

```diff
@@ -0,0 +1,190 @@
+        those defined by the parameters :gmt-term:`COLOR_BACKGROUND`,
+        :gmt-term:`COLOR_FOREGROUND`, and :gmt-term:`COLOR_NAN`]. Use
+        ``background='i'`` to match the colors for the lowest and highest
+        values in the input (instead of the output) CPT.
+    color_model :
+        [**R**\|\ **r**\|\ **h**\|\ **c**][**+c**\ [*label*]].
+        Force output CPT to be written with r/g/b codes, gray-scale values or
+        color name (**R**, default) or r/g/b codes only (**r**), or h-s-v codes
+        (**h**), or c/m/y/k codes (**c**).  Optionally or alternatively, append
+        **+c** to write discrete palettes in categorical format. If *label* is
+        appended then we create labels for each category to be used when the
+        CPT is plotted. The *label* may be a comma-separated list of category
+        names (you can skip a category by not giving a name), or give
+        *start*\[-], where we automatically build monotonically increasing
+        labels from *start* (a single letter or an integer). Append ``-`` to
+        build ranges *start*-*start+1* instead.
+    nlevels : bool or int or str
+        Set to ``True`` to create a linear color table by using the grid
+        z-range as the new  limits in the CPT. Alternatively, set *nlevels*
+        and to resample the color table into nlevels equidistant slices.
+    series : list or str
+        [*min/max/inc*\ [**+b**\|\ **l**\|\ **n**\]|\ *file*\|\ *list*\].
+        Defines the range of the new CPT by giving the lowest and highest
+        z-value (and optionally an interval). If this is not given, the
+        existing range in the master CPT will be used intact. The values
+        produced defines the color slice boundaries.  If **+n** is used it
+        refers to the number of such boundaries and not the number of slices.
+        For details on array creation, see
+        :gmt-docs:`makecpt.html#generate-1d-array`.
+    truncate : list or str
+        *zlo/zhi*.
+        Truncate the incoming CPT so that the lowest and highest z-levels are
+        to *zlo* and *zhi*. If one of these equal NaN then we leave that end of
+        the CPT alone. The truncation takes place before any resampling. See
+        also :gmt-docs:`cookbook/features.html#manipulating-cpts`.
+    output : str
+        Optional argument to set the file name with extension .cpt to store
+        the generated CPT file. If not given or False (default), saves the CPT
+        as the session current CPT.
+    reverse : str
+        Set this to True or c [Default] to reverse the sense of color
+        progression in the master CPT. Set this to z to reverse the sign of
+        z-values in the color table. Note that this change of z-direction
+        happens before *truncate* and *series* values are used so the latter
+        must be compatible with the changed *z*-range. See also
+        :gmt-docs:`cookbook/features.html#manipulating-cpts`.
+    overrule_bg :
```

Comment:
```suggestion
    overrule_bg : str
```
Seems like this is missing in `makecpt` too at https://github.com/GenericMappingTools/pygmt/blob/5b80c64bb97a6b84c5d8dcfc539ca1b097c2a5aa/pygmt/src/makecpt.py#L121, could you update the docstring there too?

Reply:
Updated for `makecpt` in e0fd4ab